schema/DanmNet.yaml

```diff
@@ -36,6 +36,12 @@ spec:
     # A Virtual Function belonging to this Physical Function is taken-up in case NetworkType is set to SRIOV.
     # OPTIONAL - STRING
     host_device: ## PARENT_DEVICE_NAME ##
+    # Name of an SR-IOV Network Device Plugin resource pool
```

Comment:
this doc also needs to be reworked to talk about device management in general, not just exclusively from SR-IOV VFs
SR-IOV is just one example of using Kubernetes devices during network provisioning

Reply:
Sorry, I have no idea what kind of other device plugin can come into the picture from networking viewpoint. I've found this list but have no clue how to refer them (except sriov) with regards DANM.
https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#examples 
On the other hand my opinion is that the device_pool attribute description should have SR-IOV specific part due to danm has specific assumptions. (eg.: one-to-one mapping)

Reply:
nope.
we are only assuming one device pool is mapped to one DanmNet. but that is the whole idea behind the generic code I made you implement:
- multiple DanmNets can refer the same Device Pool
- one Device pool can contain pretty much any number of devices, related to multiple physical "stuff"

that is not 1:1 mapping!

The device_pool parameter _generally_ represents the base resource name of the Kubernetes Devices connected to this network.
_Specifically_ for DanmNets with NetworkType: sriov this option is mandatory, and it represents a pool of VFs Pods connecting to this network will get on provisioned.

You see the difference, I hope. Just because we only have one supported use-case of a feature _right now_ , the feature, and the interfaces themselves shall be still generic, and easily re-usable  in the future.
Hence you work on this review for a week now.

Hope it was a good lesson for future endeavours :)